dytype/test_dytype.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+assert_unspecified() {
+  if grep --quiet ': #<unspecified>' <(./dytype "$@"); then
+    printf '.'
+  else
+    echo
+    echo "Expected"
+    echo "  $@"
+    echo "to be"
+    echo "  : unspecified"
+    echo "but it was"
+    echo "  $(./dytype "$@")"
```

Comment:
Why re-run? Don't we just want to save it off the result from the first run in a variable?